circle.yml

```diff
@@ -111,7 +111,15 @@ database:
     - 'echo "Skipping database phase (already ran)."'
 test:
   override:
-    - case $CIRCLE_NODE_INDEX in 0) bundle exec rake lint:javascript circle:run_tests ;; *) bundle exec rake circle:run_ui_tests ;; esac:
+    - bundle exec rake lint:javascript:
+        parallel: true
+        timeout:
+            7200
+    - bundle exec rake circle:run_tests:
+        parallel: true
+        timeout:
+            7200
+    - bundle exec rake circle:run_ui_tests:
```

Comment:
This means eyes, ui tests and lint will be run in parallel with each other, correct? This seems right, just confirming.